app/graphql/mutations/digital_object/update_descriptive_metadata.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Mutations
+  module DigitalObject
+    class UpdateDescriptiveMetadata < Mutations::BaseMutation
+      argument :id, ID, required: true
+      argument :descriptive_metadata, GraphQL::Types::JSON, required: true
+      argument :identifiers, [String], required: true
+
+      field :digital_object, Types::DigitalObjectInterface, null: false
+      field :errors, [Types::Errors::FieldedInput], null: false
+
+      def resolve(id:, descriptive_metadata:, identifiers:)
+        digital_object = ::DigitalObject::Base.find(id)
+        ability.authorize! :update, digital_object
+        digital_object.assign_attributes(
+          'identifiers' => Set.new(identifiers),
```

Comment:
```suggestion
          'identifiers' => identifiers,
```
Same note here about not needing to convert identifiers to a Set.